plugins/chain/plugin.cpp

```diff
@@ -347,30 +353,34 @@ namespace chain {
         try {
             ilog("Opening shared memory from ${path}", ("path", my->shared_memory_dir.generic_string()));
             my->db.open(data_dir, my->shared_memory_dir, STEEMIT_INIT_SUPPLY, my->shared_memory_size, chainbase::database::read_write/*, my->validate_invariants*/ );
-            auto head_block_log = my->db.get_block_log().head();
-            my->replay |= head_block_log && my->db.revision() != head_block_log->block_num();
-
-            if (my->replay) {
-                my->replay_db(data_dir, my->force_replay);
-            }
```

Comment:
Return back lines 350-355. 
They were used for replaying from command line.

I requested only to remove the checking of changed structure in this place, which you added.

Reply:
Is it ok?
my->open...
if (my->replay) {
  my->replay_db...
}
Isn't better?
if (my->replay) {
  my->replay_db...
} else {
  my->open...
}

Reply:
It'll be:
my->open...
if (my->replay) {
my->replay_db...
}
Isn't better?
if (my->replay) {
my->replay_db...
} else {
my->open...
}

Reply:
No, it is not better.
The database should be open before replay.

Please, check how it works.